src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,81 @@
+/* eslint-disable jsx-a11y/no-static-element-interactions */
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+/* eslint-disable jsx-a11y/control-has-associated-label */
```

Comment:
as I see from the comments you've resolved it in `CustomSelector`
can you avoid it here as well?